File: Dante1/Module1.07/Zadanie7.03/main.rs
fn circumference(number: f32) -> f32
{
    if number <= 0.0
    {
        return -1.0
    }
    number * std::f32::consts::PI
}

fn main()
{
    let mut input: String = String::new();
    println!("Enter: ");

    std::io::stdin().read_line(&mut input).expect("Incorrect input");
    let number: f32 = input.trim().parse().expect("Incorrect number");

    println!("Result is: {}", circumference(number));
}

File: Dante1/Module1.07/Zadanie7.03/main.c
#include <stdio.h>
#include <math.h>

float circumference(float b);

int main()
{
    float number;

    printf("Enter radix: ");
    int check = scanf("%f",&number);
    if(check != 1)
    {
        printf("Incorrect input\n");
        return 1;
    }

    float result = circumference(number);
    if(result == -1)
    {
        printf("Incorrect input\n");
        return 1;
    }
    printf("Result is: %.4f",result);
    return 0;
}

float circumference(float b)
{
    if(b <= 0)
    {
        return -1;
    }
    return b * M_PI;
}
